ios_widget_catalogue_compare/ios/Runner/OverlaySwiftUIView.swift

```diff
@@ -0,0 +1,39 @@
+import SwiftUI
+
+/// The SwiftUI view that appears as an overlay to our Flutter view.
+/// Given iOS 13 availability.
```

Comment:
13 or 14?

Reply:
I'll remove the line since it's redundant